lib/libp2p_port.ex

```diff
@@ -224,8 +214,18 @@ defmodule LambdaEthereumConsensus.Libp2pPort do
   ########################
 
   @impl GenServer
-  def init(args) do
-    {new_peer_handler, args} = Keyword.pop(args, :new_peer_handler, nil)
+  def init(_args) do
+    config = Application.fetch_env!(:lambda_ethereum_consensus, :discovery)
```

Comment:
This change makes testing more difficult since it'd need to modify global state. More specifically, the `libp2p_port` unit tests will fail.